interview-questions.md

```diff
@@ -8,62 +8,83 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: A parameter can server as the guidelines for a function when defining it. The parameters tell the function what to do with variable values inside the function
 
-  Researched answer:
+  Researched answer: A parameter is listed as a variable inside the () of a function when defining it. A argument is the value given to the () to run in place of the parameter variable.
 
 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
+  Your answer:  .map takes predefined parameters to initiate its built in iteration. Function and Current Value are required parameters, the optional parameters are Index, Array, or This Value.
 
-  Researched answer:
+  Researched answer: .map syntax is typically : array.map(function(currentValue, index, arr), thisValue). The only required parameters being Function and Current Value. 
```

Comment:
With ES6, you will often find people will use arrow syntax for .map instead of using the keyword function.  This will look like: `array.map((currentValue, index, arr) => currentValue)`